sdk/core/Azure.Core.Amqp/src/Shared/AmqpAnnotatedMessageConverter.cs

```diff
@@ -0,0 +1,956 @@
+                    message.Properties.ContentType = source.Properties.ContentType.Value;
+                }
+
+                if (source.Properties.CorrelationId != null)
+                {
+                    message.Properties.CorrelationId = new AmqpMessageId(source.Properties.CorrelationId.ToString());
+                }
+
+                if (source.Properties.CreationTime.HasValue)
+                {
+                    message.Properties.CreationTime = source.Properties.CreationTime;
+                }
+
+                if (!string.IsNullOrEmpty(source.Properties.GroupId))
+                {
+                    message.Properties.GroupId = source.Properties.GroupId;
+                }
+
+                if (source.Properties.GroupSequence.HasValue)
+                {
+                    message.Properties.GroupSequence = source.Properties.GroupSequence;
+                }
+
+                if (source.Properties.MessageId != null)
+                {
+                    message.Properties.MessageId = new AmqpMessageId(source.Properties.MessageId.ToString());
+                }
+
+                if (source.Properties.ReplyTo != null)
+                {
+                    message.Properties.ReplyTo = new AmqpAddress(source.Properties.ReplyTo.ToString());
+                }
+
+                if (!string.IsNullOrEmpty(source.Properties.ReplyToGroupId))
+                {
+                    message.Properties.ReplyToGroupId = source.Properties.ReplyToGroupId;
+                }
+
+                if (!string.IsNullOrEmpty(source.Properties.Subject))
+                {
+                    message.Properties.Subject = source.Properties.Subject;
+                }
+
+                if (source.Properties.To != null)
+                {
+                    message.Properties.To = new AmqpAddress(source.Properties.To.ToString());
+                }
+
+                if (source.Properties.UserId != null)
+                {
+                    message.Properties.UserId = source.Properties.UserId;
+                }
+            }
+
+            // Application Properties
+
+            if ((source.Sections & SectionFlag.ApplicationProperties) > 0)
+            {
+                foreach (var pair in source.ApplicationProperties.Map)
+                {
+                    if (TryCreateNetPropertyFromAmqpProperty(pair.Value, out var propertyValue))
+                    {
+                        message.ApplicationProperties[pair.Key.ToString()] = propertyValue;
+                    }
+                }
+            }
+
+            // Message Annotations
+
+            if ((source.Sections & SectionFlag.MessageAnnotations) > 0)
+            {
+                foreach (var pair in source.MessageAnnotations.Map)
+                {
+                    if (TryCreateNetPropertyFromAmqpProperty(pair.Value, out var propertyValue))
+                    {
+                        // if (SystemPropertyDateTimeKeys.Contains(pair.Key.ToString()))
```

Comment:
Is this intended to be commented out?   Should we consider removing it?

Reply:
These were removed already.